mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -44,8 +37,16 @@ export class EmployeeDetailsEventsList extends Component<EmployeeDetailsEventsLi
         return (
                 <View style={eventsContainerFlattened} key={item.calendarEventId}>
                     <View style={eventRow}>
-                        <ApplicationIcon name={this.eventTypeToGlyphIcon.get(item.type)} style={eventIcon} />
-                        <StyledText style={eventTitle}>{item.type} starts on {item.dates.startDate.format(eventDialogTextDateFormat)} and completes on {item.dates.endDate.format(eventDialogTextDateFormat)} ({item.status})</StyledText>
+                    <View style={eventLeftIconsTiny}>
+                        <View style={eventTypeIconContainerTiny}>
+                            <ApplicationIcon name={eventTypeToGlyphIcon.get(item.type)} style={eventIcon} />
+                        </View>
+                    </View>
+                        <View style={{ flex: 5 }}>
+                            <StyledText style={eventTitle}>{this.props.employeeName}</StyledText>
```

Comment:
As I understood, employee-details-events-list and employee-details-pending-requests-list are the same, expect
```
<StyledText style={eventTitle}>{this.props.employeeName}</StyledText>
<StyledText style={eventDetails}>{item.descriptionStatus}</StyledText>
<StyledText style={eventDetails}>{item.descriptionFromTo}</StyledText>
```
lets create one employee-details-events-list component, and pass that peace of StyledTexts through the its props.children